public/docs/_examples/tutorial/dart/.gitignore

```diff
@@ -0,0 +1,22 @@
+# Files and directories created by pub
```

Comment:
I suppose I never noticed because I don't normally see dotfiles. I'll check in the angular author channel.